packages/math-input/src/store/echo-reducer.ts

```diff
@@ -11,9 +14,14 @@ const initialEchoState = {
     echoes: [],
 } as const;
 
-const echoReducer = function (state = initialEchoState, action): EchoState {
+type Action = PressKeyAction | RemoveEchoAction;
+
+const echoReducer = function (
+    state: EchoState = initialEchoState,
+    action: Action,
+): EchoState {
     switch (action.type) {
-        case "PressKey":
+        case PressKeyActionType:
```

Comment:
Given that the `Action` type already defines which action this reducer supports, I think that we can stick with string constants here because only the set of supported string constant action types will be allowed in this `switch`.

Reply:
Done